shared/cfg/machines.cfg

```diff
@@ -46,3 +46,15 @@ variants:
         # Currently no USB support
         usbs =
         usb_devices =
+    - s390:
+        only s390
```

Comment:
I found that machine type must be s390-ccw-virtio here, because this value is same with `machine s390-ccw-virtio  \` in qemu command line.